components/endpoint-configuration-management/org.wso2.carbon.identity.api.server.endpoint.configuration.management/org.wso2.carbon.identity.api.server.endpoint.configuration.management.common/src/main/java/org/wso2/carbon/identity/api/server/endpoint/configuration/management/common/EndpointConfigurationManagementConstants.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.endpoint.configuration.management.common;
+
+
+/**
+ * Endpoint configuration management related constant class.
+ */
+public class EndpointConfigurationManagementConstants {
+
+    public static final String ERROR_PREFIX = "NSM-";
+
+    public static final String CONFIGURATION_CONTEXT_PATH = "/endpoint-configurations";
+    public static final String CONFIGURATION_RESOURCE_TYPE = "endpoint-configuration";
+    public static final String PLUS = "+";
+    public static final String URL_ENCODED_SPACE = "%20";
+    public static final String RESOURCE_NOT_EXISTS_ERROR_CODE = "CONFIGM_00017";
```

Comment:
What is the difference between this error code and the rest?

Reply:
This is to verify that resource is exist in configuration store